dev/KaairaGupta/calibration_plot/calibration_plot.py

```diff
@@ -0,0 +1,38 @@
+from sklearn.calibration import calibration_curve as calib_c
+import matplotlib.pyplot as plt
+import matplotlib.lines as mlines
+import matplotlib.transforms as mtransforms
+
+def calibration_plot(clf_list, clf_names, test_data, labels_test, n_bins):
+    """
+        Input:
+            clf_list: list of objects of a classifier which has method clf.predict_proba(test_data) with output y = list of predicted prbablities for two classes <np.array<np.array[2]>>
+            test_data: input for clf.predict(test_data)
+            test_data_labels: correct labels for test data
+            n_bins: number of bins
+            clf_names: list of names clf in clf_list
+        Output: Plots a calibartion plot displaying the observed occurrence rates vs the bin midpoints for each mode
+    """
+
+    cc_list = []
+
+    for clf in clf_list:
+        pred_p = clf.predict_proba(test_data)
+        y, x = calib_c(labels_test, pred_p[:,1], n_bins=10)
```

Comment:
nit: please don't clobber single function import names. Just leave `calibration_curve` as `calibration_curve` for easy readability.

Reply:
I have resolved this.